helix-view/src/editor.rs

```diff
@@ -1083,8 +1082,7 @@ pub struct Editor {
     pub diagnostics: Diagnostics,
     pub diff_providers: DiffProviderRegistry,
 
-    pub debugger: Option<dap::Client>,
-    pub debugger_events: SelectAll<UnboundedReceiverStream<dap::Payload>>,
+    pub dap_servers: dap::registry::Registry,
```

Comment:
```suggestion
    pub debug_adapters: dap::registry::Registry,
```

to match `language_servers`